src/main/groovy/io/gatling/gradle/GatlingPlugin.groovy

```diff
@@ -98,13 +98,7 @@ final class GatlingPlugin implements Plugin<Project> {
             group = "Gatling"
 
             if (simulationFQN) {
-                simulations = {
-                    include(
-                        "${simulationFQN.replace('.', '/')}.java",
-                        "${simulationFQN.replace('.', '/')}.scala",
-                        "${simulationFQN.replace('.', '/')}.kt"
-                    )
-                }
+                simulationClass = simulationFQN
```

Comment:
So... no condition?

Reply:
I don't get it.
This class name is turned into a filter, cf https://github.com/gatling/gatling-gradle-plugin/pull/99/files#diff-4a9f5862b5a900fb63e18d537d80dc750764c83b0b749b0ae9cfb9afa1a8f3b4R89

Reply:
```diff
-if (simulationFQN) {
-  simulationClass = simulationFQN
-}
+simulationClass = simulationFQN
```

If null we won't assign it, but it was null... so it won't hurt. Or do I miss something else?

Reply:
Ahhh. No, it can't hurt.